docs/userGuide/syntax/links.mbdf

```diff
@@ -30,6 +30,27 @@ MarkBind home is at [here][1].
 
 Links to files of the generated site (e.g., an HTML page or an image file) can be specified either as relative paths or absolute paths.
 
+****Auto-conversion of extension****
+
+<div class="indented">
+
+Do indicate markdown files with its original extension (**.md**) as it will automatically be converted to a html extension (**.html**) when the site is generated. 
```

Comment:
```suggestion
You may link to markdown files using its original extension (**.md**) as it will automatically be converted to a html extension (**.html**) when the site is generated. 
```